lib/user_agent/operating_systems/darwin.rb

```diff
@@ -55,7 +55,11 @@ module Darwin
         '21.2.0' => '15.2',   # also 15.2.1
         '21.3.0' => '15.3',   # also 15.3.1
         '21.4.0' => '15.4',   # also 15.4.1
-        '21.5.0' => '15.5',
+        '21.5.0' => '15.5',   # also 15.5.1
```

Comment:
```suggestion
        '21.5.0' => '15.5',
```
Nitpick: the comment `# also 15.5.1` doesn't seem right. From the [table](https://www.theiphonewiki.com/wiki/Kernel#iOS.2FiPadOS) (`iOS/iPadOS`) there is no `15.5.1` being mapped. `15.5.1` mapping is for `tvOS`.